bonfire/openshift.py

```diff
@@ -327,3 +328,18 @@ def get_reservation(name=None, namespace=None, requester=None):
             return None
 
     return None
+
+
+def get_pool_size_limit(pool):
+    pool_data = get_json("namespacepool", name=pool)
+
+    return int(pool_data["spec"].get("sizeLimit", None)) if pool_data else None
```

Comment:
I think you want to return `0` here instead of `None` so that the integer comparison back in bonfire.py doesn't fail:

```
    return int(pool_data["spec"].get("sizeLimit", 0)) if pool_data else 0
```